src/BaselineOfIDE.package/BaselineOfIDE.class/instance/groups..st

```diff
@@ -209,7 +209,6 @@ spec group: 'Shout-Group' with: #(
 ).
 
 spec group: 'MetacelloExtras-Core' with: #(
-	'Metacello-GitBasedRepository'	
```

Comment:
This is wrong. Somehow the merge is removing this line and this line should stay.